packages/eds-core-react/src/components/Popover/Popover.tsx

```diff
@@ -125,47 +140,42 @@ export const Popover = forwardRef<HTMLDivElement, PopoverProps>(
       [floating, ref],
     )
 
-    useEffect(() => {
-      if (refs.reference.current && refs.floating.current && open) {
-        return autoUpdate(refs.reference.current, refs.floating.current, update)
-      }
-    }, [refs.reference, refs.floating, update, open])
```

Comment:
Note that `refs` and `update` are no longer used and should be removed from useFloating (line 115-116)